ci-operator/step-registry/ipi/conf/azure/oidc-creds-provision/ipi-conf-azure-oidc-creds-provision-ref.yaml

```diff
@@ -7,6 +7,9 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  dependencies:
+    - name: "release:latest"
+      env: RELEASE_IMAGE_LATEST_FROM_BUILD_FARM
```

Comment:
I am curious where did you reference the workaround?

Reply:
@jianlinliu When I saw this problem with azure, I created a cluster with [gcp Workload Identity using clusterbot](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-gcp-modern/1681562927326302208) and didn't notice the problem.  So I went to reference aws and gcp and realized it was an image issue. 

For the workaround, I found it here: https://github.com/search?q=repo%3Aopenshift%2Frelease%20RELEASE_IMAGE_LATEST_FROM_BUILD_FARM&type=code and then I took a trip to learn how to use it.